preprocessy/outliers/handleoutlier.py

```diff
@@ -0,0 +1,100 @@
+class HandleOutlier:
+
+
+    """ Class for handling outliers on its own or according to users needs.
+
+        Private methods
+       _ _ _ _ _ _ _ _ _ _
+
+       __returnQuartiles() : returns the 5% and 95% mark in the distrubution 
+                             of data(Values above are default values)
+
+        Public Methods
+       _ _ _ _ _ _ _ _ _
+
+       handleOutliers() : Takes in the dataset as input, finds the quartiles
+                          and returns the dataset within the interquartile 
+                          range. Function run only on int64 and float64 
+                          specified columns.
+
+    """
+
+
+    def __init__(self, train_df, params):
+
+
+        """ Function to initialize a few parameters used to make the process
+            run without human interaction.
+
+            Parameters to be entered by the user include :
+                -The dataset
+                -cols: Any specific columns in dataset the user wants to 
+                       remove outliers from. If not entered columns will be
+                       selected based on their dtype(int and float only) and 
+                       outliers will be removed from them
+                -removeoutliers : default = True
+                -replace : default = False => if user wants to replace 
+                           outliers with -999 everywhere instead of 
+                           removing them
+                -q1 : specify the starting range (Default is 0.05)
+                -q3 : specify the end of the range (Default is 0.95)
+
+        """
+        self.train_df = train_df
+        self.cols = []
+        self.removeoutliers = True
+        self.replace = False
+        self.quartiles = {}
+        self.qone = 0.05
```

Comment:
Variable could be renamed to `first_quartile`.

Reply:
Will correct